core/src/Authorization.html

```diff
@@ -104,26 +125,28 @@
 
         const logoutCallback = () => {
           const onLogoutFn = LuigiConfig.getConfigValue('auth.events.onLogout');
-          if (onLogoutFn) { onLogoutFn(); };
+          if (onLogoutFn) {
+            onLogoutFn();
+          }
 
           this.set({ isLoggedIn: false });
           localStorage.removeItem('luigi.auth');
         };
 
-        const customLogoutFn = LuigiConfig.getConfigValue(`auth.${LuigiConfig.getConfigValue('auth.use')}.logoutFn`);
+        const customLogoutFn = LuigiConfig.getConfigValue(
+          `auth.${LuigiConfig.getConfigValue('auth.use')}.logoutFn`
+        );
         if (isFunction(customLogoutFn)) {
           customLogoutFn(idpProviderInstance.settings, authData, logoutCallback);
-        }
-        else if (isFunction(idpProviderInstance.logout)) {
+        } else if (isFunction(idpProviderInstance.logout)) {
           idpProviderInstance.logout(authData, logoutCallback);
-        }
-        else {
+        } else {
           logoutCallback();
           window.location.href = idpProviderInstance.settings.logoutUrl;
-        };
-      },
+        }
+      }
     },
-    components: {},
+    components: {}
     // store: () => store
```

Comment:
Feel free to remove obsolete things when cleaning up. Guess this `store` is there since ages and never used.